src/content/en/updates/2018/01/paintapi.md

```diff
@@ -0,0 +1,278 @@
+    for(let y = 0; y < geom.height/size; y++) {
+      for(let x = 0; x < geom.width/size; x++) {
+        ctx.fillStyle = colors[(x + y) % colors.length];
+        ctx.beginPath();
+        ctx.rect(x*(size + spacing), y*(size + spacing), size, size);
+        ctx.fill();
+      }
+    }
+  }
+}
+
+registerPaint('checkerboard', CheckerboardPainter);
+```
+
+Now we can use the same code for all different kind of checkerboards. But even
+better, we can now go into DevTools and [fiddle with the
+values](https://googlechromelabs.github.io/houdini-samples/paint-worklet/parameter-checkerboard/)
+until we find the right look.
+
+<div style="display: flex; justify-content: center">
+  <video loop muted controls>
+    <source
+      src="https://storage.googleapis.com/webfundamentals-assets/paintapi/checkercast_vp8.webm"
+      type="video/webm; codecs=vp8">
+    <source
+      src="https://storage.googleapis.com/webfundamentals-assets/paintapi/checkercast_x264.mp4"
+      type="video/mp4; codecs=h264">
+  </video>
+</div>
+
+Note: It would be great to parameterize the colors, too, wouldn’t it? The spec
+allows for the `paint()` function to take a list of arguments. This feature is
+not implemented in Chrome yet, as it heavily relies on Houdini’s Properties &
+Values API, which still needs some work before it can ship.
+
+## Browsers that doesn’t support paint worklet
+At the time of writing, only Chrome has paint worklet implemented. While there
+are positive signals from all other browser vendors, there isn’t much progress.
+To keep up to date, check [Is Houdini Ready Yet?](https://ishoudinireadyyet.com)
+on a regular basis. In the meantime, you should make sure to use Progressive
+Enhancement to keep your code running even if there’s no support for paint
+worklet. To make sure things work as expected, you have to adjust your code in
+two places: The CSS and the JS.
+
+Detecting support for paint worklet in JS can be done by checking the `CSS` object:
+
+    if ('paintWorklet' in CSS) {
+      CSS.paintWorklet.addModule('mystuff.js');
+    }
+
+For the CSS side, you have two options. You can use `@supports`:
+
+    @supports (background: paint(id)) {
+      /* ... */
+    }
+
+A more compact trick is to use the fact that CSS invalidates and subsequently
+ignores an entire property declaration if there is an unknown function in it. If
+you specify a property twice — first without paint worklet, and then with the
+paint worklet — you get progressive enhancement:
+
+    textarea {
+      background-image: linear-gradient(0, red, blue);
+      background-image: paint(myGradient, red, blue).
+    }
+
+In browsers _with_ support for paint worklet, the second declaration of
+`background-image` will overwrite the first one. In browsers _without_ support
+for paint worklet, the second declaration is invalid and will be discarded,
+leaving the first declaration in effect.
+
+## Use cases
+There’s many use-cases for Paint Worklets, some of them more obvious than
```

Comment:
Change 'There's' to 'There're' or 'There are'.
'use-case' > 'use case'. Please fix throughout.